sdjournal/read.go

```diff
@@ -256,6 +256,48 @@ process:
 	}
 }
 
+// SkipN skips the next n entries and returns the number of skipped entries and an eventual error.
+func (r *JournalReader) SkipN(n int) (int, error) {
+	var i int
+	for i := 1; i <= n; i++ {
+		c, err := r.journal.Next()
+		if err != nil {
+			return i, err
+		} else if c == 0 {
+			return i, nil
+		}
+	}
+	return i, nil
+}
+
+// FollowTail synchronously follows the JournalReader, writing each new journal entry to entries.
+// It will start from the next unread entry.
+func (r *JournalReader) FollowTail(entries chan *JournalEntry) error {
```

Comment:
I'm not overly positive with this proposed signature, as it lacks at least a `Context` to cancel the inner looping logic.
Additionally, I'm wondering whether it is better to return two channels (`entries` and `err`) so that the whole follow-tailing logic can be handled asynchronously.